modules/tinymce/src/plugins/lists/main/ts/core/Delete.ts

```diff
@@ -260,6 +263,13 @@ const backspaceDelete = (editor: Editor, isForward: boolean): boolean => {
 };
 
 const setup = (editor: Editor): void => {
+  editor.on('ExecCommand', (e) => {
+    const cmd = e.command.toLocaleLowerCase();
+    if ((cmd === 'delete' || cmd === 'forwarddelete') && hasListSelection(editor)) {
```

Comment:
Should this run if the command has been prevented? I guess it shouldn't hurt.

Reply:
I don't think we should prevent it here since the command has already been executed at that point it's not like it's preventing the `ExecCommand` from executing that is the `BeforeExecCommand` event. I guess the event name here is a bit missleading maybe they should have been `ExecCommand` and `AfterExecCommand` instead not sure.

Reply:
Yeah I realised this was what happens after, it was more just in case someone wanted to prevent this for whatever reason. It's an extreme edge case though so more than happy to leave it as is, I just thought it was worth asking.